cafy_pytest/plugin.py

```diff
@@ -236,6 +236,10 @@ def pytest_addoption(parser):
     group.addoption('--cafygta', dest='cafygta', action='store_true',
                     help='Variable to enable cafy gta, default is False')
 
+    group = parser.getgroup('Generate all.log.html - Enable/Disable')
```

Comment:
This is a bit of a specific group for CLI args. Also, every added arg makes it harder for users to understand the help output - is this really necessary? Could it be added as an ini config option instead?